src/main/scala/cromwell/logging/WorkflowLogger.scala

```diff
@@ -0,0 +1,80 @@
+package cromwell.logging
+
+import akka.event.LoggingAdapter
+import cromwell.engine.WorkflowDescriptor
+import org.slf4j.Logger
+
+/**
+ * This sets up two loggers for a given workflow, specified by the
+ * WorkflowDescriptor.
+ *
+ * 1) the Akka event logger (optional).  The log messages sent to the Akka
+ *    logger will propagate to the root logger (see logback.xml)
+ *
+ * 2) A logger that writes to a file that only has log messages
+ *    for this particular workflow.  This makes it easier to debug
+ *    a particular workflow if the messages aren't interleaved with
+ *    other workflows.  This logger does NOT propagate to the root
+ *    logger
+ */
+case class WorkflowLogger[T](descriptor: WorkflowDescriptor, akkaLogger: Option[LoggingAdapter] = None, otherLoggers: Seq[Logger] = Seq.empty[Logger]) {
```

Comment:
It's nice that this is parameterized by the WorkflowDescriptor, we should be able to remove all the workflow ID tagging code from all callers and centralize that here to prevent issues like DSDEEPB-1810.